src/structures/gdt.rs

```diff
@@ -312,6 +312,13 @@ impl Descriptor {
     }
 
     /// Creates a TSS system descriptor for the given TSS.
+    ///
+    /// While it is possible to create multiple Descriptors that point to the
+    /// same TSS, this generally isn't recommended, as the TSS usually contains
+    /// per-CPU information such as the RSP and IST pointers. Instead, there
+    /// should be exactly one TSS and one corresponding TSS Descriptor per CPU.
+    /// Then, each of these descriptors should be placed in a GDT (which can
+    /// either be global or per-CPU).
```

Comment:
We will no longer support shared GDTs with the next breaking release (#323).

Reply:
For this 0.14 documentation, should we still mention that global GDTs?

I think it would still be reasonable to mention that GDTs _could_ be global. Global GDTs are supported by the SDM. 

For 0.15 it might be reasonable to expand on the this comment. Something like: "while GDTs can be global, this library only supports per-cpu GDTs".

Reply:
> For this 0.14 documentation, do you think the documentation I have is OK?

Yes.

> 
> I think it's reasonable to mention that GDTs _could_ be global. Global GDTs are supported by the SDM.
> 
> For 0.15 it might be reasonable to expand on the this comment. Something like: "while GDTs can be global, this library only supports per-cpu GDTs".

Sounds good to me!